### ISSUE TITLE
[FEATURE REQUEST] when finish Action defined in settings

### ISSUE THREAD

**Issue by devildant** [RandomEngy/VidCoder#1309]
### Feature details

Hi,
it could be useful to be able to define the default action once the encoding is finished queue/task.

for example when I launch vidcoder, it automatically launches the encodings it has in queue, however, I would also like the action once the queue is finished to be automatically set to close vidcoder or restart

**Comment by RandomEngy** [comment]
There's a "when done" option that can automatically close VidCoder or restart when either the current job or queue is complete. Are you saying you'd like an option to remember the last picked choice? I had it clear on each launch to avoid people's computers unexpectedly shutting down.

**Comment by devildant** [comment]
Yes it's that. An option for remember that at startup, with an options in settings

**Comment by RandomEngy** [comment]
I've added this option in Global Options -> Advanced in v12.0 Beta.
